src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.scss

```diff
@@ -206,7 +206,7 @@ p {
   }
 
   input.mat-input-element:enabled {
-    background-color: #ffffff !important;
+    background-color: #fffee6 !important;

```

Comment:
😢Yeah, I hope someday we can encourge business to follow the standard. But this seems like what they want.

Reply:
I am too hoping